vars/checkChangeInDevelopment.groovy

```diff
@@ -0,0 +1,89 @@
+import com.sap.piper.GitUtils
+import groovy.transform.Field
+
+import com.sap.piper.ConfigurationMerger
+import com.sap.piper.cm.ChangeManagement
+import com.sap.piper.cm.ChangeManagementException
+
+@Field def STEP_NAME = 'checkChangeInDevelopment'
+
+@Field Set parameterKeys = [
+    'changeDocumentId',
+    'cmClientOpts',
+    'credentialsId',
+    'endpoint',
+    'failIfStatusIsNotInDevelopment',
+    'git_from',
+    'git_to',
+    'git_label',
+    'git_format'
+  ]
+
+@Field Set stepConfigurationKeys = [
+    'changeDocumentId',
```

Comment:
Could you change it to changeId like the other steps?

Reply:
I vote for changing the other occurrences to ```changeDocumentId```this this is more precise. The term change is not that clear. The official term used inside SolutionManager is "Changedocument"

Reply:
see PR #174